### PR TITLE
add theme, section and blog notes cards

### PR DESCRIPTION
**Pull request by erenjaegar77**

### CODE REVIEW

File: src/components/Home/LatestArticles/LatestArticles.tsx [hunk 1]
```diff
@@ -4,42 +4,88 @@ import { Inner } from '../../Common/Inner';
...
+        <Date>{date}</Date>
+      </CardContent>
+    </Card>
   );
 });
 
 const ArticleList = styled.div`
   display: grid;
   grid-template-columns: repeat(4, 1fr);
+  column-gap: ${theme.spacing2};
```
Comment (ReflectiveChimp) [inline src/components/Home/LatestArticles/LatestArticles.tsx:76]: design is 24px gap

File: src/components/Home/LatestArticles/LatestArticles.tsx [hunk 2]
```diff
@@ -4,42 +4,88 @@ import { Inner } from '../../Common/Inner';
...
-const Outer = styled.div``;
+const ImageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  border-bottom: 2px solid ${theme.cardBorder};
+`;
+
+const Image = styled(GatsbyImage)`
+  maxwidht: 200px;
+  maxheight: 64px;
```
Comment (ReflectiveChimp) [inline src/components/Home/LatestArticles/LatestArticles.tsx:18]: `max-width` and `max-height`
they can be removed as gatsby image component has aspect ratio set.

File: src/components/Common/Section/Section.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { SectionSubTitle } from './components/SectionSubTitle';
+
+type SectionProps = {
+  title: string;
+  subtitle?: string;
+  children: ReactNode;
+};
+
+const Outer = styled.section`
+  height: 50vh;
```
Comment (ReflectiveChimp) [inline src/components/Common/Section/Section.tsx:15]: Remove

File: src/components/Common/Layout/Layout.tsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ const Wrapper = styled.div`
   grid-template-columns: minmax(0, 1fr);
   min-height: 100vh;
   width: 100%;
+  background: linear-gradient(108.4deg, #2a2b42 0%, #141520 100%);
```
Comment (ReflectiveChimp) [inline src/components/Common/Layout/Layout.tsx:14]: This should probably go on `body` in the common styles.

**Review by ReflectiveChimp — CHANGES_REQUESTED** [review]
I'm not sure how 'common' the `Section` component will be, since it is only for home page.
Maybe worth moving there.

**Comment by dotpablocom** [comment]
Hi

**Pull request merged**